src/components/hover-card.vue

```diff
@@ -109,22 +109,23 @@ $border-radius: 12px;
 .hover-card { width: 290px; }
 
 .hover-card-heading {
+  $icon-margin-top: 2px;
+
   background-color: #666;
   border-inline: $border;
   border-top: $border;
   border-top-left-radius: $border-radius;
   border-top-right-radius: $border-radius;
   color: #fff;
   display: flex;
-  // Align the top of the text with the top of the icon.
-  line-height: 1;
-  padding: $padding-panel-body;
-  padding-bottom: 12px;
+  line-height: 1.2;
```

Comment:
This is still less than the default, but it's a number that we use in several places in Frontend.